deploy/helm-chart/README.md

```diff
@@ -0,0 +1,135 @@
+# NGINX Kubernetes Gateway Helm Chart
+
+## Introduction
+
+This chart deploys the NGINX Kubernetes Gateway in your Kubernetes cluster.
+
+## Prerequisites
+
+- [Helm 3.0+](https://helm.sh/docs/intro/install/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+> Note: NGINX Kubernetes Gateway can only run in the `nginx-gateway` namespace. This limitation will be addressed in
+the future releases.
+
+## Installing the Chart
+
+> Note: The Gateway API resources from the standard channel (the CRDs and the validating webhook) must be installed
+before deploying NGINX Kubernetes Gateway. By default, they will be installed by the Chart if not already
+present in the cluster. If they are already installed in your cluster, please ensure they are the correct version as
+supported by the NGINX Kubernetes Gateway - [see the Technical Specifications](../../README.md#technical-specifications).
+Helm will not upgrade CRDs - to do so manually, see
+[Upgrading the Gateway API resources](#upgrading-the-gateway-resources).
+
+### Installing the Chart from the OCI registry
+
+To install the chart with the release name `my-release` (`my-release` is the name that you choose) into the
+nginx-gateway namespace (with optional `--create-namespace` flag - you can omit if the namespace already exists), and
+the Gateway API resources from the standard channel (if not already present):
+
+```
+helm install my-release oci://ghcr.io/nginxinc/charts/nginx-gateway --version 0.0.0-edge --create-namespace --wait-for-jobs -n nginx-gateway
```

Comment:
`--wait-for-jobs` help says `if set and --wait enabled, will wait until all Jobs have been completed before marking the release as successful.`. Does this mean we also need the `--wait` flag? What's the behavior without that flag?

Reply:
Good catch! This was a typo, I've been running with both flags locally. I've tested it out, without the `--wait` it returns successfully when the NKG deployment is is running but not yet ready (so it waits for the validating web hook but not the NKG pod readiness), with the `wait` it returns only when everything is ready (pods and services - so I think in the case of LoadBalancer service only when an external IP is assigned). Hmm, this doesn't really correspond with the docs, but this behaviour is probably what we want?

Reply:
Seems like having both flags is the better gate for readiness, right?

Reply:
I've added it